packages/manager/modules/pci/src/projects/project/storages/databases/database/users/add/add.routing.js

```diff
@@ -1,3 +1,5 @@
+import isFeatureActivated from '../../../features.constants';
```

Comment:
This should be driven by API, it is too much of Product logic to be harcoded

Reply:
Maybe the name should be changed, as it suggest that we do feature flipping, which we are not. This is just an array that says which page should be shown per engine, it is only frontend logic. We put all the logic in this place to avoid boilerplate code with many if on each page.